traveltimepy/dto/transportation.py

```diff
@@ -42,3 +42,23 @@ class PublicTransport(BaseModel):
     pt_change_delay: Optional[int] = None
     walking_time: Optional[int] = None
     max_changes: Optional[MaxChanges] = None
+
+
+class DrivingFerry(BaseModel):
```

Comment:
it was already implemented in Ferry. You just needed to add CyclingPublicTransport

Reply:
removed CyclingFerry and DrivingFerry